lib/movie.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class Movie < Item
+  def initialize(silet)
+    super()
+    # Boolean
+    @silet = silet
```

Comment:
Also your (source) parameter should be private, because you will be modifying it through your Source Class and when you add a movie there, you need to be able to automatically push the Source object into the @items of your Movie object.

For this you need to create a different setter method within your class, like this:
```
def source=(source)
    @source = source
    source.items << self unless source.items.include?(self)
end
```
 
 Because you will be setting it through your class method. you are keeping the @source parameter private. Also you only need an `attr_reader` for you @silent instance.